temba/channels/models.py

```diff
@@ -1143,6 +1158,34 @@ def send_facebook_message(cls, channel, msg, text):
                                response_status=response.status_code)
 
     @classmethod
+    def send_line_message(cls, channel, msg, text):
+        from temba.msgs.models import Msg, WIRED
+
+        channel_access_token = channel.config.get(Channel.CONFIG_AUTH_TOKEN)
+
+        data = json.dumps({'to': msg.urn_path, 'messages': [{'type': 'text', 'text': text}]})
+
+        start = time.time()
+        headers = {'Content-Type': 'application/json', 'Authorization': 'Bearer %s' % channel_access_token}
+        headers.update(TEMBA_HEADERS)
+        response = requests.post('https://api.line.me/v2/bot/message/push',
+                                 data=data,
+                                 headers=headers)
+        content = json.loads(response.content)
+
+        if response.status_code == 200:
+            Msg.mark_sent(channel.config['r'], channel, msg, WIRED, time.time() - start, external_id=None)
+
+            ChannelLog.log_success(msg=msg,
+                                   description="Successfully delivered",
+                                   method='POST',
+                                   url=response.request.url,
+                                   response=response.content,
+                                   response_status=response.status_code)
+        else:
+            ChannelLog.log_error(msg=msg, description=content.get('message'))
```

Comment:
When logging errors the pattern is generally to put all the information possible to help in debugging. IE, the URL, response, content etc.. (this should be in the log_success) as well